### ISSUE TITLE
[Kobo glo] [Low Memory Crash] Recommendation of maximum cache memory or cache proportion settings

### ISSUE THREAD

**Issue by Tao4free** [koreader/koreader#9186]
Hello guys,
I am using kobo glo to read some pdfs with 25 mb size.
It will crashed after I read some pages the crash screenshot is below.

Is there any recommendation of settings that can avoid the crashing.

My current changed setting.
DGLOBAL_CACHE_SIZE_MAXIMUM = 1024*1024*60

![161654731308_ pic](https://user-images.githubusercontent.com/29458813/172734690-d5e0286c-c0aa-4475-9fee-c87410fde6c4.jpg)

**Comment by poire-z** [comment]
`DGLOBAL_CACHE_SIZE_MAXIMUM = 1024*1024*60`

No real idea. Pinging @NiLuJe if these negative free RAM size and percentage in the log in the picture make sense to you :)

**Comment by NiLuJe** [comment]
It's a tilde (`~`), not a minus sign ;).

TL;DR: Leave `DGLOBAL_CACHE_SIZE_MAXIMUM` alone, the actual cache size is computed at runtime based on actually available memory (enable debug logging to see the results in the logs at boot).

Other than that, it's probably a case of the usual "aww, shucks, not enough RAM on your device" ;).

**Comment by Tao4free** [comment]
Thanks, I am also finding some new machines for better pdf reading experience.
However, kobo haven't release some large memory machine, basically 512m maximum 1G.

Thanks for the comments.

**Comment by NiLuJe** [comment]
The Elipsa has 1G of RAM ;).

**Comment by Tao4free** [comment]
It's a shame they didn't provide more than 2G according to the price of Elipsa :(
No perfect device from Kobo or other ereader provider.

Thanks anyway.
